src/backend/src/controllers/change-requests.controllers.ts

```diff
@@ -76,8 +71,101 @@ export const reviewChangeRequest = async (req: Request, res: Response) => {
         approved: true
       }
     });
-  }
+    const wbs = await prisma.wBS_Element.findUnique({
+      where: {
+        wbsElementId: foundCR.wbsElementId
+      },
+      include: {
+        workPackage: true,
+        project: true
+      }
+    });
+
+    if (!wbs) {
+      return res.status(404).json({ message: `wbs element with id #${foundCR.wbsElementId} not found` });
+    }
+
+    const wp = { ...wbs.workPackage };
+
+    const proj = { ...wbs.project };
+
+    if (!wp && proj) {
+      let updatedBudgetImpact;
+      if (!proj.budget) {
```

Comment:
i think budget should always exist

Reply:
For some reason its telling me that budget is an optional value so I just changed it to force unwrap?

Reply:
hm it works on my local so not sure. and if you look at the schema budget isn't optional so idk